website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -114,28 +114,30 @@ For testing, use your feature branch in cumulus-orca github repo and `ORCA-test-
 
 While running the `Deploy Dev RDS Stack` stage, replace the following variables with yours. This is because some variables are sensitive and some will vary depending upon the user running the pipeline. Hitting 'play' next to `Deploy Dev RDS Stack` and `Deploy Dev Cumulus and ORCA Stack` brings up a checkbox list to run multiple jobs at once. Note that none of the checkboxes should be checked.
 
-- AWS_ACCESS_KEY_ID
-- AWS_SECRET_ACCESS_KEY
+- AWS_ACCESS_KEY_ID(for cumulus sandbox account)
+- AWS_SECRET_ACCESS_KEY(for cumulus sandbox account)
 - PREFIX
-- AWS_ACCOUNT_ID
+- AWS_ACCOUNT_ID(for cumulus sandbox account)
 - DB_ADMIN_PASSWORD
 - DB_USER_PASSWORD
 
-Make sure the proper buckets, dynamoDB table and the EC2 key pair are first created manually using the same prefix before running the pipeline. See this cumulus [documentation](https://nasa.github.io/cumulus/docs/deployment/deployment-readme) and [deployment with cumulus documentation](https://github.com/nasa/cumulus-orca/blob/develop/website/docs/developer/deployment-guide/deployment-with-cumulus.md#modifying-cumulus-tfterraformtfvars) for additional details. Typically, these are the buckets that need to be created first
+The ORCA buckets and dynamoDB table are created automatically in bamboo `Deploy Dev Cumulus and ORCA Stack` stage. However, these can also be created manually if desired by the user. These are the buckets that need to be created:
+
 - `<PREFIX>-internal`
 - `<PREFIX>-level0`
 - `<PREFIX>-public`
 - `<PREFIX>-private`
 - `<PREFIX>-protected`
 - `<PREFIX>-orca-primary`
 - `<PREFIX>-tf-state` (for storing the terraform state file)
+- `<PREFIX>-orca-archive-worm`
+- `<PREFIX>-orca-reports`

```

Comment:
There is no differentiation in this list between the buckets that are automatically created, and the buckets that must be manually created. Recommend having info on the DR OU and Cumulus OU buckets in separate steps.

Reply:
currently, all the buckets are automatically created. 

Updated the doc to separate/organize the bucket deployment steps.

Reply:
Per @AndrewDorn  comment this section is still a little confusing to me  and I feel we could better separate this to make it clear.

Reply:
Yes updated the paragraphs

Reply:
They're all upgraded to automatic deployment? Where is, say, the level0 bucket created?

Reply:
yes correct. Level0 is created in cumulus sandbox account. 
See this commit 
https://github.com/nasa/cumulus-orca/pull/297/commits/c805b131cf00f228a71c93c5bc2359b2079b12e7

Reply:
yes it was created in the previous PR. This template creates the buckets https://github.com/nasa/cumulus-orca/blob/develop/integration_test/buckets.tf.template